File: src/test/programmers/level1/CreateWeirdCharacters.java
package test.programmers.level1;

import java.util.Locale;

public class CreateWeirdCharacters {
    public static void main(String[] args) {
        System.out.println(solution("try hello world"));
    }
    static String solution(String s) {
        String answer = "";
        String[] strings = s.split("");
        int idx = 0;

        for(String str : strings) {
            idx = str.equals(" ") ? 0 : idx + 1;

            answer += idx % 2 == 0 ? str.toLowerCase() : str.toUpperCase();
        }

        return answer;
    }
}

File: src/test/programmers/python/level1/CreateWeirdCharacters.py
def solution(s):
    a = s.split(" ")
    for i in range(len(a)):
        char = list(a[i])
        for j in range(len(char)):
            if j % 2 == 0:
                char[j] = char[j].upper()
            else:
                char[j] = char[j].lower()
        a[i] = "".join(char)
    return " ".join(a)
